### ISSUE TITLE
Add support for Neo Geo Pocket (Color)

### ISSUE THREAD

**Issue by ducalex** [ducalex/retro-go#108]
**Describe the problem you're trying to solve**
I want to play Neo Geo Pocket games

**Describe the solution you have in mind**
Likely port the RACE or NeoPOP emulator.

xrip has a port running on the pico here: https://github.com/xrip/pico-neogeo-pocket

**Additional context**
None

**Comment by dilixi** [comment]
Hi, does this can be accomplished on esp32?

@ducalex

**Comment by ducalex** [comment]
Yes I am fairly confident it can be accomplished on the esp32 but someone will have to do the work to port @xrip's work.

I am not currently working on that myself, if that was the question!
